# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/streams/idlharness-shadowrealm.window.js.ini
# Commit: 2ce86a76d99d
# Full Hash: 2ce86a76d99dc03cb8f88799680824bb4fcb0fa0
# Author: Tom Schuster <evilpies@gmail.com>
# Date: 2022-12-01 16:18:29
# Regressor Bug: 1734244
# File Overlap Count: 1
# Description:
#   Bug 1734244 - Implement async iteration of ReadableStream. r=mgaudet,peterv
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D153312
# ==============================================================================

diff -r c353f81b2823 -r 2ce86a76d99d testing/web-platform/meta/streams/idlharness-shadowrealm.window.js.ini
--- a/testing/web-platform/meta/streams/idlharness-shadowrealm.window.js.ini	Thu Dec 01 08:51:47 2022 +0000
+++ b/testing/web-platform/meta/streams/idlharness-shadowrealm.window.js.ini	Thu Dec 01 09:04:34 2022 +0000
@@ -1,6 +1,2 @@
 [idlharness-shadowrealm.window.html]
   prefs: [javascript.options.experimental.shadow_realms:true]
-
-  [ReadableStream interface: async iterable<any>]
-    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1734244
-    expected: FAIL